examples/Textual-Responses.ipynb

```diff
@@ -0,0 +1,337 @@
+              "https://tlk.s3.yandex.net/annotation_tasks/librispeech/test-clean/2615.mp3    it is evident therefore that the present trend...\n",
+              "https://tlk.s3.yandex.net/annotation_tasks/librispeech/test-clean/2616.mp3    it is also noticeable that the serviceability ...\n",
+              "https://tlk.s3.yandex.net/annotation_tasks/librispeech/test-clean/2617.mp3    consumption becomes a larger element in the st...\n",
+              "https://tlk.s3.yandex.net/annotation_tasks/librispeech/test-clean/2618.mp3    among the country population its place is to s...\n",
+              "https://tlk.s3.yandex.net/annotation_tasks/librispeech/test-clean/2619.mp3    the result is a great mobility of the labor em...\n",
+              "Name: true_label, Length: 2620, dtype: object"
+            ]
+          },
+          "execution_count": 9,
+          "metadata": {},
+          "output_type": "execute_result"
+        }
+      ],
+      "source": [
+        "gt"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "aYPa2ZDniEw5"
+      },
+      "source": [
+        "Now let's see how we can solve the aggregation problem with text responses. We will use three methods - ROVER, TextRASA, and TextHRRASA. All these methods solve the same problem - text aggregation - but in different ways. \n",
+        "Let's import them"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 10,
+      "metadata": {
+        "id": "RBEHhOUwYBj9"
+      },
+      "outputs": [],
+      "source": [
+        "from crowdkit.aggregation import ROVER\n",
+        "from crowdkit.aggregation import TextHRRASA\n",
+        "from crowdkit.aggregation import TextRASA"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "_-MUkakUlBFi"
+      },
+      "source": [
+        "At first, we will use Recognizer Output Voting Error Reduction - ROVER. It's a dynamic programming method to align sequences. We need to determine the tokenizer and detokenizer functions. For the tokenizer, we will split a sentence by spaces and for the detokenizer we will glue the words into a string"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "CGy2nWqceYaL"
+      },
+      "outputs": [],
+      "source": [
+        "tokenizer = lambda s: s.split(' ')\n",
+        "detokenizer = lambda tokens: ' '.join(tokens)\n",
+        "\n",
+        "result_ROVER = ROVER(tokenizer, detokenizer).fit_predict(df)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "8nl9JNhw-PKj"
+      },
+      "source": [
+        "Next, we will try TextHRRASA. We need to determine the encoder - a callable that takes a text and returns a NumPy array containing the corresponding embedding. For that, we will use the model from sentence_transformers. The model.encode returns the embedding"
```

Comment:
HRRASA is a more advanced version of RASA. It would make more sense if we first used RASA and then HRRASA

Reply:
What does HRRASA do? How it's different from ROVER. Also say that ROVER can construct new transcriptions and RASA/HRRASA can only extract one from provided